docker_build/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# An integration test & dev container which builds and installs cuDF from main
+ARG CUDA_VERSION=10.2
+ARG CUDA_SHORT_VERSION=${CUDA_VERSION}
+
+FROM nvidia/cuda:${CUDA_VERSION}-devel
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/lib
+# Needed for cudf.concat(), avoids "OSError: library nvvm not found"
+ENV NUMBAPRO_NVVM=/usr/local/cuda/nvvm/lib64/libnvvm.so
+ENV NUMBAPRO_LIBDEVICE=/usr/local/cuda/nvvm/libdevice/
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV CC=/usr/bin/gcc
+ENV CXX=/usr/bin/g++
```

Comment:
Are these actually needed? I imagine they would be the default anyway.

Reply:
Not needed, removed them.